### PR TITLE
Update catalog export tool for Tiger and clean MD integration

### PR DESCRIPTION
**Pull request by michalvanco**
Please review by individual commits.

Completely removes the usage of old MD service from UI.SDK.

Last missing part was to adapt the catalog-export tool.

JIRA: RAIL-2822

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1881/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3502/ : SUCCESS in 10m 50s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1882/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3503/ : FAILURE in 2m 35s

File: tools/catalog-export/src/loaders/tiger/tigerDateDatasets.ts [hunk 1]
```diff
@@ -1,35 +1,30 @@
...
     getReferencedDataset,
     LabelMap,
-    TagMap,
 } from "./tigerCommon";
 
 type DatasetWithAttributes = {
-    dataset: DatasetResourceSchema;
-    attributes: AttributeResourceSchema[];
+    dataset: DatasetsItem;
+    attributes: AttributesItem[];
```
Comment (no23reason) [inline tools/catalog-export/src/loaders/tiger/tigerDateDatasets.ts:16]: Here you replaced `AttributeResourceSchema[]` by `AttributesItem[]` but on line 20, you replaced it by `Attributes`. This should probably stay consistent, right?

Comment (michalvanco) [reply tools/catalog-export/src/loaders/tiger/tigerDateDatasets.ts:16]: it's valid as the returned type is `Attributes` and next logic iterates on the objects inside (`AttributesItem`)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1883/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3504/ : SUCCESS in 15m 51s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/607/ : SUCCESS in 28s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/868/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3507/ : SUCCESS in 21m 23s

**Pull request merged**